src/graphic/helper/roundSector.ts

```diff
@@ -89,6 +82,48 @@ function computeCornerTangents(
     };
 }
 
+function calcCircleCenter(x: number, y: number, r: number, angle: number) {
+    return {
+        x: x + r * mathCos(angle),
+        y: y + r * mathSin(angle)
+    };
+}
+
+// For compatibility, don't use normalizeCssArray
+// 5 represents [5, 5, 5, 5]
+// [5] represents [5, 5, 0, 0]
+// [5, 10] represents [5, 5, 10, 10]
+// [5, 10, 15] represents [5, 10, 15, 15]
+// [5, 10, 15, 20] represents [5, 10, 15, 20]
+function normalizeCornerRadius(
+    cr: number | string | (number | string)[],
+    r0: number,
+    r: number
+) {
+    let arr: (number | string)[];
+    if (isArray(cr)) {
+        const len = cr.length;
+        if (len === 4) {
+            arr = cr;
+        }
+        else if (len === 3) {
+            arr = cr.concat(cr[len - 1]);
+        }
+        else if (len === 2) {
+            arr = [cr[0], cr[0], cr[1], cr[1]];
+        }
+        else {
+            arr = [cr[0], cr[0], 0, 0];
+        }
+    }
+    else {
```

Comment:
Is it possible for `cr` to be `undefined`?

Reply:
There is a check before this normalization.